client/post-editor/editor-deprecation-dialog/style.scss

```diff
@@ -0,0 +1,94 @@
+
+.editor-deprecation-dialog {
+
+	&.components-modal__frame {
+		border: none;
+	}
+
+	.components-modal__content {
+		@include breakpoint( '>660px' ) {
```

Comment:
If we use the `$break-small` mixin from Gutenberg we can use the same breakpoints as Gutenberg

Reply:
Good idea, done in 50a7122a07.